pages/ksphere/kommander/1.1/install-airgapped/index.md

```diff
@@ -0,0 +1,128 @@
+
+# Naming
+
+Across this document, some terms are used which are defined here:
+* management cluster - konvoy cluster that is going to be running kommander
+* attached cluster - cluster, be it konvoy or non-konvoy one, which are going to be attached to the management cluster
+* docker registry - a registry from which all clusters download docker images used by installation
+
+# Before you begin
+
+Before installing, verify that your environment meets the following basic requirements:
+
+* a docker registry should already contain all the docker images required for installation and this includes also kommander images. The `konvoy_air_gapped.tar.bz2` tarball will contain the required artifacts.
+
+* connectivity with clusters that are going to be attached to the management
+  cluster:
+  * both management and attached clusters should be able to connect to the docker registry
+  * management cluster should be able to connect to attached cluster's API server
+
+* all the prerequisites outlined in [airgapped konvoy installation][https://docs.d2iq.com/ksphere/konvoy/1.6/install/install-airgapped/#before-you-begin] in case of konvoy clusters. For attached clusters that are 3rd party clusters, please refer to the vendor's documentation.
+
+## Control plane nodes
+
+Control plane nodes of Konvoy clusters should meet the minimal requirements outlined in [airgapped konvoy installation][https://docs.d2iq.com/ksphere/konvoy/1.6/install/install-airgapped/#control-plane-nodes]. For 3rd-party clusters control plance requirements refer to the vendor specification.
+
+## Worker nodes
+
+Worker nodes should meet the minimal requirements outlined in [airgapped konvoy installation][https://docs.d2iq.com/ksphere/konvoy/1.6/install/install-airgapped/#worker-nodes].
+
+## Operating system and services for all nodes
+
+All nodes should meet the same minimal requirements outlined in [airgapped konvoy installation][https://docs.d2iq.com/ksphere/konvoy/1.6/install/install-airgapped/#operating-system-and-services-for-all-nodes].
+
+## Defininig inventory file
+
+Installing airgapped Kommander does not require any additional changes in the `inventory.yaml` file.
+
+# Configure the Kubernetes cluster
+
+The `cluster.yaml` file provides the configuration details for creating your Konvoy cluster. Installing Kommander in an airgapped environment requires some additional configuration, which is described below. It is assumed that all the `cluster.yaml` changes outlined in [airgapped konvoy installation][https://docs.d2iq.com/ksphere/konvoy/1.6/install/install-airgapped/#configure-the-image-registry] documentation were also applied.
+
+First of all, Kommander needs to use self-hosted charts repo that will be running on top of Konvoy cluster, as it will not be able to connect to the default one through the public Internet.
```

Comment:
```suggestion
1.  Ensure Kommander can use the self-hosted charts repo running on top of the Konvoy cluster. It can not connect to the default one through the public Internet.
```